maven-plugins/helidon-maven-plugin/src/main/java/io/helidon/build/maven/GraalNativeMojo.java

```diff
@@ -168,6 +169,18 @@ public class GraalNativeMojo extends AbstractMojo {
     @Parameter(defaultValue = "false", property = "native.image.skip")
     private boolean skipNativeImage;
 
+    /**
+     * Module name for {@code --module} argument.
+     */
+    @Parameter(property = "native.image.module")
+    private String module;
+
+    /**
+     * Module path for {@code --module-path} argument.
+     */
+    @Parameter()
+    private List<String> modulePath;
```

Comment:
Is this practical ? We should derive the module-path dynamically like we do for the class-path (see `getClasspath()`)

Reply:
See also https://github.com/codehaus-plexus/plexus-languages/tree/master/plexus-java/src/main/java/org/codehaus/plexus/languages/java/jpms

Reply:
See it used here: https://github.com/apache/maven-javadoc-plugin/blob/master/src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java#L5171

Reply:
Thanks for the links !